internal/shim-sgx/src/heap.rs

```diff
@@ -133,23 +116,25 @@ where
             Some(page) => page,
             None => return self.pos(),
         };
-
-        if old < new {
-            for page in old..new {
-                if self.is_allocated(page) {
+        match old.cmp(&new) {
+            Ordering::Less => {
+                let region = Region::new(
+                    Address::new(old * Page::SIZE),
+                    Address::new(new * Page::SIZE),
+                );
+                if self.ledger.map(region, Access::empty()).is_err() {
```

Comment:
why `Access::empty()` ?

Reply:
They don't really matter before SGX2 is stable but does not hurt to set them right, so I'll update this.